astropy/modeling/core.py

```diff
@@ -3616,6 +3617,62 @@ def fix_inputs(modelinstance, values):
     return CompoundModel('fix_inputs', modelinstance, values)
 
 
+class _TieParameterTo(Callable):
```

Comment:
Can you move the `_TieParameterTo` class to `modeling/utils.py`? `modeling/core.py` already has a lot in it. This class looks to be independent of anything in this module and acts as a utility for `tie_inputs` it makes sense to put it with the rest of the utility classes, such as  `_BoundingBox`.

Reply:
Good idea. Done!